@AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageFCRDraw.sqf

```diff
@@ -172,6 +172,7 @@ private _pointsArray = [];
         _unitSelAndWpnStatus = ["ANTS"];
     };
     private _ident = (["FCR",_unitType,_unitStatus] + _unitSelAndWpnStatus) joinString "_";
+    systemChat format ["Ident = %1", _ident];
```

Comment:
Please can you remove this `systemChat`?

Reply:
Fixed